template/user.email.update/headers.prop

```diff
@@ -0,0 +1,2 @@
+In-Reply-To: <workitems/{{.workitem.Data.ID}}@notify.openshift.io>
```

Comment:
No workitem in the auth context. None of these headers make any sense.. ?

Reply:
removed it :)